### DISCUSSION TITLE
[SUGGESTION] Reduce retry effects

### DISCUSSION THREAD

**Discussion by RaiLTC** [ppy/osu#29038] [Ideas]
Accidental retry button presses (playing close to the retry button key causes that) is extremely distracting, ESPECIALLY on short activation time, the entire screen is almost blacked out and the audio is muffled instantaneously while the map is still playing

Stable has slight blackout effect and no audio muffling (not that I noticed in the last 10 years at least), which is not that noticable when hitting the retry key by accident

**Comment by peppy** [comment]
Have you considered moving your keys a bit? 😅

**Reply by RaiLTC** [reply]
Yeah, but I always end up hitting that key somehow lol
It might be more relevant considering keyboard sizes are being reduced lately (rip numpad my beloved)
